chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -398,7 +402,17 @@ EXVAL=0
 export HOME=/root
 
 IP="<%= @admin_ip %>"
-HTTP_SERVER="<%= @admin_ip %>:<%= @web_port %>"
+<% if @ip_version == "6" -%>
+IPV6=1
+<% else  -%>
+IPV6=0
```

Comment:
Below where it is first used? I'm so confused.

Reply:
yeah it is confusing, but really the code here is run first because it's outside of functions. Then it's used in functions. So was just following the pattern that was previously used for the $IP. As I figured once this is templated out into a bash script I wanted it to look like the old version as how knows in anyone ever goes in an hacks it. I guess they shouldn't.

I was lazy, so figured following an existing pattern and minimalise the change to the crowbar_join rather then refactor and probably cause or introduce bugs.